src/main/webapp/js/instructorFeedbackEdit/customFeedbackPaths.js

```diff
@@ -0,0 +1,358 @@
+    $('.form_question').on('change', '.participantSelect', function() {
+        var $questionForm = $(this).closest('.form_question');
+        var $container = $questionForm.find('.custom-feedback-paths-spreadsheet');
+        var $giverSelect = $questionForm.find('select[id^="' + FEEDBACK_QUESTION_GIVERTYPE + '"]');
+        var $recipientSelect = $questionForm.find('select[id^="' + FEEDBACK_QUESTION_RECIPIENTTYPE + '"]');
+        var giverType = $giverSelect.val();
+        var recipientType = $recipientSelect.val();
+        
+        if (giverType === FEEDBACK_PARTICIPANT_TYPE_CUSTOM
+                || recipientType === FEEDBACK_PARTICIPANT_TYPE_CUSTOM) {
+            $giverSelect.find('option[value="' + FEEDBACK_PARTICIPANT_TYPE_CUSTOM + '"]').remove();
+            $recipientSelect.find('option[value="' + FEEDBACK_PARTICIPANT_TYPE_CUSTOM + '"]').remove();
+        }
+        
+        giverType = $giverSelect.val();
+        recipientType = $recipientSelect.val();
+        updateFeedbackPathsSpreadsheet($container, giverType, recipientType);
+    });
+    
+    $('.form_question').on('click', '.add-rows-button', function() {
+        var $questionForm = $(this).closest('.form_question');
+        var $container = $questionForm.find('.custom-feedback-paths-spreadsheet');
+        var numRowsToAdd = parseInt($questionForm.find('.add-rows-input').val());
+        var hotInstance = $container.handsontable('getInstance');
+        if (numRowsToAdd > 0) {
+            hotInstance.alter('insert_row', hotInstance.countRows(), numRowsToAdd);
+        }
+    });
+    
+    $('.form_question').on('click', '.customize-button', function() {
+        var $questionForm = $(this).closest('.form_question');
+        $questionForm.find('div[class*="numberOfEntitiesElements"]').hide();
+        $questionForm.find('.participantSelect').each(function() {
+            if ($(this).val() !== FEEDBACK_PARTICIPANT_TYPE_CUSTOM) {
+                var $customFeedbackParticipantTypeOption =
+                        $('<option></option>').attr('value', FEEDBACK_PARTICIPANT_TYPE_CUSTOM)
+                                              .text('Custom');
+                $(this).append($customFeedbackParticipantTypeOption)
+                       .val(FEEDBACK_PARTICIPANT_TYPE_CUSTOM);
+            }
+        });
+        enableAllVisibilityOptionsRowsForQuestion($questionForm);
+        var columns = getColumnsForFeedbackPathsSpreadsheet(FEEDBACK_PARTICIPANT_TYPE_CUSTOM,
+                                                            FEEDBACK_PARTICIPANT_TYPE_CUSTOM);
+        var $container = $questionForm.find('.custom-feedback-paths-spreadsheet');
+        var hotInstance = $container.handsontable('getInstance');
+        hotInstance.updateSettings({
+            columns: columns
+        });
+        hotInstance.validateCells();
+    });
+}
+
+function enableAllVisibilityOptionsRowsForQuestion($questionForm) {
+    enableRow($questionForm.children(':first'), 1);
```

Comment:
Maybe we can use the `enableAllRows()` in `visibilityOptions.js` in #5803. Since it is not merged yet, can just copy the function to `visibilityOptions.js`